remote/rpc/clientV10.go

```diff
@@ -0,0 +1,47 @@
+package rpc
+
+import (
+	"context"
+	"io"
+	"net/rpc"
+
+	"github.com/weaveworks/flux/api/v10"
+	"github.com/weaveworks/flux/api/v6"
+	"github.com/weaveworks/flux/remote"
+)
+
+// RPCClientV10 is the rpc-backed implementation of a server, for
```

Comment:
Bit of copypasta in this comment? (referring to v8; if anything, it ought to refer to v9)